django_mongodb/introspection.py

```diff
@@ -1,6 +1,31 @@
 from django.db.backends.base.introspection import BaseDatabaseIntrospection
+from django.db.models import Index
 
 
 class DatabaseIntrospection(BaseDatabaseIntrospection):
+    ORDER_DIR = {1: "ASC", -1: "DESC"}
+
     def table_names(self, cursor=None, include_views=False):
         return sorted([x["name"] for x in self.connection.database.list_collections()])
+
+    def get_constraints(self, cursor, table_name):
+        indexes = self.connection.database[table_name].index_information()
```

Comment:
Could we use `self.connection.get_collection` instead of `.database[table_name]`. But I don't know if we have to keep get_collection.